ext/CreateParametersExt.jl

```diff
@@ -175,19 +175,21 @@ function FarquharParameters(
         :s6 => FT(328.15),
         :E => FT(0.05),
     )
-
-    if maximum(is_c3) > 1
-        error(
-            "is_c3 has maximum of $(maximum(is_c3)). is_c3 should be between 0 and 1",
-        )
-    end
-    if minimum(is_c3) < 0
-        error(
-            "is_c3 has minimum of $(minimum(is_c3)). is_c3 should be between 0 and 1",
-        )
-    end
+    #=
```

Comment:
Temporarily commenting this out because values over the ocean are not set, but then they do not satisfy the checks here (\in [0,1])